.github/workflows/automerge.yml

```
@@ -0,0 +1,29 @@
+name: Auto merge
+
+on:
+  pull_request:
+    branches:
+      master
+  pull_request_review:
+    types:
+      - submitted
+  check_suite:
+    types:
+      - completed
+  status: {}
+jobs:
+  # Automatically merge approved and green dependabot PRs.
+  auto-merge-dependabot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: pascalgn/automerge-action@v0.8.4
+        if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
```

IIRC, we've migrated all dependabots to v2.

```suggestion
        if: github.actor == 'dependabot[bot]'
```